### PR TITLE
App header

### PR DESCRIPTION
**Pull request by mahdiAbu3li**

### CODE REVIEW

File: src/components/AppHeader/AppHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+import AttachMoneyIcon from "@material-ui/icons/AttachMoney";
+import SettingsIcon from "@material-ui/icons/Settings";
+import TrendingUpIcon from "@material-ui/icons/TrendingUp";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import KitchenIcon from "@material-ui/icons/Kitchen";
+import TurnedInIcon from "@material-ui/icons/TurnedIn";
+import ExitToAppIcon from "@material-ui/icons/ExitToApp";
+import Avatar from "@material-ui/core/Avatar";
+import { useStyles } from "./AppHeaderStyle";
+  const AppHeader =  () =>  {
```
Comment (Abdallatif) [inline src/components/AppHeader/AppHeader.tsx:17]: Add a line before the component and remove the indentation

File: src/components/AppHeader/AppHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+              <Button
+                color="inherit"
+                className={classes.barButton}
+                startIcon={<KitchenIcon />}
+              >
+                Product
+              </Button>
+
+              <Button
+                aria-controls="simple-menu"
```
Comment (Abdallatif) [inline src/components/AppHeader/AppHeader.tsx:55]: fix the menu name

File: src/components/AppHeader/AppHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                aria-haspopup="true"
+                onClick={handleClick}
+                color="inherit"
+                className={classes.barButton}
+                startIcon={<GroupIcon />}
+              >
+                people
```
Comment (Abdallatif) [inline src/components/AppHeader/AppHeader.tsx:62]: you may need to capitalize this as the rest of the items

File: src/components/AppHeader/AppHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+                <ExpandMoreIcon />
+              </Button>
+              <Menu
+                id="simple-menu"
+                anchorEl={anchorEl}
+                keepMounted
+                open={Boolean(anchorEl)}
+                onClose={handleClose}
+              >
+                <MenuItem onClick={handleClose}>item1</MenuItem>
```
Comment (Abdallatif) [inline src/components/AppHeader/AppHeader.tsx:72]: item1?

File: src/components/AppHeader/AppHeaderStyle.ts [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+export const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
+    },
+    menuButton: {
+      marginRight: theme.spacing(2),
+    },
+    title: {
+      color: "black",
```
Comment (Abdallatif) [inline src/components/AppHeader/AppHeaderStyle.ts:12]: use the `theme` to define colors

File: src/components/AppHeader/AppHeaderStyle.ts [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    },
+    barButton: {
+      paddingLeft: "0",
+      color: "white",
+      fontWeight: "bold",
+      
+      margin: "8px",
```
Comment (Abdallatif) [inline src/components/AppHeader/AppHeaderStyle.ts:19]:
```suggestion
      margin: theme.spacing(1),
```

File: src/components/AppHeader/AppHeaderStyle.ts [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+        color: "#d50000",
+      },
+    },
+    barList: {
+      display: "flex",
+      justifyContent: "center",
+      marginLeft: "10px",
+    },
+    bar: {
+      backgroundColor: "#ffa726",
```
Comment (Abdallatif) [inline src/components/AppHeader/AppHeaderStyle.ts:30]: use the `theme` to define colors

File: src/components/AppHeader/AppHeaderStyle.ts [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+    },
+    avatar: {
+      marginRight: "10px",
+      marginTop: "10px",
+    },
+    barIcon: {
+      color: "#ffecb3",
+      fontWeight: "bold",
+      fontSize: "1rem",
+      fontFamily: '"Sansita Swashed", cursive',
```
Comment (Abdallatif) [inline src/components/AppHeader/AppHeaderStyle.ts:54]: why are you changing the `fontFamily` here?

**Review by Abdallatif — CHANGES_REQUESTED** [review]

File: src/components/AppHeader/AppHeader.tsx (hunk 4 above)
Comment (mahdiAbu3li) [reply src/components/AppHeader/AppHeader.tsx:72]: i dont know what inside the menu !

Comment (Abdallatif) [reply src/components/AppHeader/AppHeader.tsx:72]: `People` menu has `Customers`,`Suppliers`
`Category` menu has `Sales`, `Expense`

File: src/components/AppHeader/AppHeaderStyle.ts (hunk 8 above)
Comment (mahdiAbu3li) [reply src/components/AppHeader/AppHeaderStyle.ts:54]: because its icon for app

**Pull request merged**